keyboards/atreus/keymaps/ridingqwerty/keymap.c

```diff
@@ -0,0 +1,98 @@
+    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  
+    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,                      _______, KC_4,    KC_5,    KC_6,    _______,
+    _______, _______, _______, _______, _______,                   _______, KC_1,    KC_2,    KC_3,    KC_BSLS,
+    _______, _______, _______, _______, _______, KC_MINS, KC_EQL,  KC_0,    KC_0,    KC_DOT,  _______, _______
+  ),
+  [_SYMBOL] = LAYOUT( /* SM_SPC */
+    KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC,                   KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN,
+    KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN,                   _______, _______, _______, _______, KC_COLN, 
+    _______, _______, _______, _______, _______,                   _______, _______, _______, _______, KC_BSLS, 
+    _______, _______, _______, _______, _______, KC_LBRC, KC_RBRC, _______, _______, _______, _______, _______
+  ),
+  [_F_KEYS] = LAYOUT( /* FK_DOT */
+    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                     KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  
+    KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,                    _______, _______, _______, _______, KC_F11,
+    KC_F11,  KC_F12,  _______, _______, _______,                   _______, _______, _______, _______, KC_F12,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+  ),
+  [_SECRET] = LAYOUT( /* shhhh... */
+    RESET,   _______, _______, RUSTY,   FUEL,                      KC_F13,  _______, _______, _______, _______,
+    AR1ST,   SYSNOC,  _______, _______, _______,                   _______, _______, _______, OS_LAB,  _______,
+    CDLOCAL, _______, C0RE,    VAXIS,   _______,                   _______, MUNKY,   _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+  ),
+  [_FINAL] = LAYOUT( /* . */
+    _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, 
+    _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+  )
+};
+
+// custom tap/hold keys
+uint16_t key_timer;
+bool process_record_keymap(uint16_t keycode, keyrecord_t *record) {
+    switch(keycode) {
+        case RG_QUOT:   
+            if (record->event.pressed) {
+                key_timer = timer_read();
+                layer_on(_NUMBER);
+                register_mods(MOD_BIT(KC_RGUI));
+            } else {
+                unregister_mods(MOD_BIT(KC_RGUI));
+                layer_off(_NUMBER);
+                if (timer_elapsed(key_timer) < TAPPING_TERM) {
+                    register_code(KC_QUOT);
```

Comment:
You could use `tap_code` instead of both of these commands

Reply:
Updated.